docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss

```diff
@@ -109,34 +101,44 @@
   }
 }
 
-@mixin hb-unordered-list {
+@mixin hb-base-list {
```

Comment:
I think I made a mistake here, actually having an extra mixin makes things more complicated as there are multiple places to set rules. Also, there aren't many shared rules, so there's no really a benefit. Please revert to previous state, sorry :/